engine/class_modules/priest/sc_priest.cpp

```diff
@@ -2349,6 +2406,7 @@ void priest_t::create_options()
   add_option( opt_bool( "priest.gathering_shadows_bug", options.gathering_shadows_bug ) );
   add_option( opt_bool( "priest.t30_multiple_bender", options.t30_multiple_bender ) );
   add_option( opt_bool( "priest.t30_yshaarj", options.t30_yshaarj ) );
+  add_option( opt_float( "priest.prayer_of_mending_bounce_rate", options.prayer_of_mending_bounce_rate ) );
```

Comment:
We probably want to add at least a min for this option since you are passing it directly into the buff duration, potentially a max as well?

Reply:
Added both min and max, with comments describing